### PR TITLE
New keymap for the DZ65RGB

### PR DESCRIPTION
**Pull request by sidcarter**
## Description

New Keymap for the dz65rgb:
* adds a layer for mac vs linux keymaps
* puts a reset on the FNM layer

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Just a heads up, if the only changes between the mac and linux layouts are the left Alt and CMD/GUI keys, there is bootmagic, that can do this. 

https://docs.qmk.fm/#/feature_bootmagic?id=keycodes

If you just want the left keys, then `MAGIC_SWAP_LALT_LGUI` and `MAGIC_UNSWAP_LALT_LGUI`.

File: keyboards/dztech/dz65rgb/keymaps/chocol8/keymap.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#include QMK_KEYBOARD_H
+
+enum tofu68_layers {
+	_MAQWERTY,
```
Comment (drashna) [inline keyboards/dztech/dz65rgb/keymaps/chocol8/keymap.c]: Indenting is off here.  

Not sure if this is intentional or due to a mix of spaces and tabs.

Comment (sidcarter) [reply keyboards/dztech/dz65rgb/keymaps/chocol8/keymap.c]: fixed in new commit(s) @drashna

**Comment by sidcarter** [comment]
> https://docs.qmk.fm/#/feature_bootmagic?id=keycodes

add the keycodes and removed the extra layer. thanks @drashna

File: keyboards/dztech/dz65rgb/keymaps/chocol8/keymap.c [hunk 2]
```diff
@@ -1,55 +1,25 @@
...
 #define FNM MO(_FNM)
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-	[_MAQWERTY] = LAYOUT_65_ansi(
+	[_QWERTY] = LAYOUT_65_ansi(
 		KC_GESC, KC_1,     KC_2,     KC_3,   KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,  KC_BSPC, KC_GRV,  \
 		KC_TAB,  KC_Q,     KC_W,     KC_E,   KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,     KC_LBRC,  KC_RBRC, KC_BSLS, KC_DEL, \
 		KC_GRV,  KC_A,     KC_S,     KC_D,   KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN,  KC_QUOT,           KC_ENT,  KC_PGUP, \
-		KC_LSPO, KC_Z,     KC_X,     KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSPC,           KC_UP,   KC_PGDN,  \
+		KC_LSPO, KC_Z,     KC_X,     KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT,           KC_UP,   KC_PGDN,  \
```
Comment (noroadsleft) [inline keyboards/dztech/dz65rgb/keymaps/chocol8/keymap.c]: `KC_LSPO` paired with a regular `KC_RSFT`? Was this change deliberate?

Comment (sidcarter) [reply keyboards/dztech/dz65rgb/keymaps/chocol8/keymap.c]: not deliberate. fixed now. thanks @noroadsleft

**Comment by noroadsleft** [comment]
Thanks!
